src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -58,4 +65,12 @@ public function itShouldHaveCodeAndName(string $code, string $name): void
             sprintf('Cannot find catalog promotions with code "%s" and name "%s" in the list', $code, $name)
         );
     }
+
+    /**
+     * @Then this catalog promotion should be usable
```

Comment:
As we have `this` in the step, maybe it would be worth checking does the code from the event is the same as the code of promotion 💃 But it can be an improvement